### DISCUSSION TITLE
mspider/intro/comp-archi-ui/architect

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2890] [Announcements]
# mspider/intro/comp-archi-ui/architect

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/comp-archi-ui/architect.html

**Comment by kebe7jun** [comment]
服务管理包含服务条目，细化一下？

**Comment by kebe7jun** [comment]
网格配置功能更丰富了。
